libkirk/kselftests.py

```diff
@@ -116,6 +116,43 @@ async def get_suites(self, sut: SUT) -> list:
 
         return ["cgroup", "bpf"]
 
+    async def find_command(self, sut: SUT, command: str) -> Test:
+        if not sut:
+            raise ValueError("SUT is None")
+
+        if not command:
+            raise ValueError("command is empty")
+
+        cmd_args = command.split(" ")
```

Comment:
@acerv will this work properly on parameters with spaces, e.g. `-X"foo bar"`?

Reply:
This is possible indeed, but i don't know how to handle it otherwise